quantum/keymap_extras/keymap_slovak.h

```diff
@@ -141,9 +141,9 @@
  */
 // Row 1
 #define SK_TILD ALGR(SK_PLUS) // ~
-#define SK_CIRC ALGR(SK_LCAR) // ^ (dead)
-#define SK_BREV ALGR(SK_SCAR) // ˘ (dead)
-#define SK_RNGA ALGR(SK_CCAR) // ° (dead)
+#define SK_BREV ALGR(SK_LCAR) // ˘ (dead)
+#define SK_CIRC ALGR(SK_SCAR) // ^ (dead)
+//#define SK_RNGA ALGR(SK_CCAR) // ° (dead)
```

Comment:
```suggestion
#define SK_CIRC ALGR(SK_SCAR) // ^ (dead)
#define SK_BREV ALGR(SK_CCAR) // ˘ (dead)
```
Per the Windows OSK:
![image](https://user-images.githubusercontent.com/4781841/150750471-72dd1748-6e09-44c8-aaf9-054ebe4dba2b.png)

Reply:
ok. how can i accept the change you proposed?

Reply:
@KraXen72 's changes are correct for the Slovak layout on Linux (https://i.imgur.com/moFZm0v.png) and it is also correct if we look at the Slovak keyboard layout shown in the official Microsoft docs: https://docs.microsoft.com/en-us/globalization/keyboards/kbdsl.html (Click on the <kbd>AltGr</kbd> key in that last website to activate the AltGr "layer"). [This](http://kbdlayout.info/KBDSL/) also confirms.

Reply:
actually, I'll have to check if this would work, i think the sk keymap was updated at some point

Reply:
That seems correct according to the Linux Slovak keyboard (https://i.imgur.com/moFZm0v.png) and according to the official Microsoft docs : https://docs.microsoft.com/en-us/globalization/keyboards/kbdsl.html (Press the <kbd>AltGr</kbd> key on this site to visualise the AltGr "layer").  [This other site](http://kbdlayout.info/KBDSL/) also confirms that <kbd>AltGr</kbd>+<kbd>š</kbd> = <kbd>^</kbd> and <kbd>AltGr</kbd>+<kbd>č</kbd> = <kbd>˘</kbd>

Reply:
If the Linux or macOS leyouts differ from Windows then they would need to be added as their own headers - refer to the other files in `keymap_extras`.

Reply:
just checked it on my machine and you are indeed correct. i thought you changed the two keys i fixed but you found another bug in the slovak keymap, and it should be the way you suggested